Classes/EventStore/Normalizer/ValueObjectNormalizer.php

```diff
@@ -21,6 +21,9 @@
  */
 final class ValueObjectNormalizer implements DenormalizerInterface
 {
+    protected $resolveConstructorMethodCache = [];
```

Comment:
```suggestion
    /**
     * @var array
     */
    private $resolveConstructorMethodCache = [];
```

Reply:
Done

Reply:
nitpick: It's still `protected` for no reason :)